### ISSUE TITLE
idea

### ISSUE THREAD

**Issue by vinniec** [jifunks/botany#39]
I saw that you also did rpg task.
why don't you merge the two projects and allow to give some water to the plants every time you complete a task?

**Comment by ryliejamesthomas** [comment]
Wow, nice idea!

**Comment by vinniec** [comment]
Yep, i think that is difficult to find  a formula that work with task and gamification
